0-bootstrap/modules/jenkins-agent/files/jenkins_gce_startup_script.sh

```diff
@@ -42,7 +42,7 @@ wget "https://releases.hashicorp.com/terraform/${tpl_TERRAFORM_VERSION}/terrafor
     rm -rf /var/lib/apt/lists/*
 
 echo "**** Startup Step 6/9: Download and install the Terraform validator ****"
-gsutil cp gs://terraform-validator/releases/2019-04-04/terraform-validator-linux-amd64 .
+gsutil cp gs://terraform-validator/releases/2020-03-05/terraform-validator-linux-amd64 .
```

Comment:
I think #107 is also updating this.

Reply:
Yes, you are right. I will leave the change here anyways. Thank you.